src/components/reusables/Header.tsx

```diff
@@ -8,11 +8,12 @@ export interface HeaderProps {
 
 export default class Header extends React.Component<HeaderProps, void> {
   render(): JSX.Element {
+    let src = this.props.imgSrc ? require(`${this.props.imgSrc}`) : "";
     return(
       <nav className="navbar navbar-default">
         <div className="navbar-header">
           <span>{this.props.text}</span>
-          <img className="navbar-brand img-rounded" alt={this.props.alt} src={require(`${this.props.imgSrc}`)} />
+          <img className="navbar-brand img-rounded" alt={this.props.alt} src={src} />
```

Comment:
You should make `src` above default to undefined and then wrap this in a conditional because you do not want to render an empty image element.